doc/user/content/ingest-data/stripe.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Stripe"
+description: "How to stream data from Stripe to Materialize using webhooks"
+menu:
+  main:
+    parent: "webhooks"
+    name: "Stripe"
+    weight: 10
+aliases:
+  - /sql/create-source/webhook/#connecting-with-stripe
+---
+
+[Stripe](https://stripe.com/) is a developer-focused payments platform. You can
+use Materialize's webhook source to ingest data from Stripe in real time.
+
+This guide will walk you through the steps to ingest data from Stripe to
+Materialize.
+
+### Before you begin
+
+Ensure that you have a Stripe account.
+
+## Step 1. (Optional) Create a Materialize cluster
+
+If you already have a cluster for your webhook sources, you can skip this step.
+
+To create a Materialize cluster, follow the steps outlined in our
+create a [cluster guide](/sql/create-cluster),
+or create a managed cluster for all your webhooks using the following SQL statement:
+
+```sql
+CREATE CLUSTER webhooks_cluster SIZE = '3xsmall';
+```
+
+## Step 2. Set up a webhook source
+
+Using the secret from the previous step, create a [webhook source](/sql/create-source/webhook/)
+in Materialize to ingest data from Stripe:
+
+```sql
+CREATE SOURCE stripe_source
+IN CLUSTER webhooks_cluster
+FROM WEBHOOK
+    BODY FORMAT JSON;
+```
+
+After a successful run, the command returns a `NOTICE` message containing the [webhook URL](https://materialize.com/docs/sql/create-source/webhook/#webhook-url).
+Copy and store it. You will need it for the next steps. Otherwise, you can query it here: [`mz_internal.mz_webhook_sources`](https://materialize.com/docs/sql/system-catalog/mz_internal/#mz_webhook_sources).
+
+## Step 3. Create a webhook endpoint in Stripe
+
+1. In Stripe, go to **Developers > Webhooks**.
+
+2. Click **Add endpoint**.
+
+3. Enter the webhook URL from the previous step in the *Endpoint URL* field.
+
+4. Configure the events you'd like to receive.
+
+5. Create the endpoint.
+
+6. Copy the signing secret and save it for the next step.
+
+## Step 4. Recreate the webhook source with validation
+
+To ensure that only your Stripe account can write to your webhook source, add
+[validation with the `CHECK` clause](/sql/create-source/webhook/#validating-requests)
+to your webhook.
+
+First, add the signing secret from the last step to Materialize:
+
+```sql
+CREATE SECRET stripe_webhook_secret AS '<signing-secret>';
+```
+
+Then recreate the webhook source with the following `CHECK` clause:
+
+```sql
+DROP SOURCE stripe_source;
```

Comment:
Ah, I put this in there because you need to clear out the `stripe_source` created in step 2. I broke this out into a dedicated code block so it's easier to see.